### PR TITLE
Small changes

### PR DESCRIPTION
**Pull request by Randium**
Now that imad has added an emoji check, I thought we could add that real quick and then have the master branch alone left.

### CODE REVIEW

**Comment by Randium** [comment]
This one's got a single conflict. Can somebody resolve it? The "conflict" is ridiculously small, and yet it's there.

File: interpretation/check.py [hunk 1]
```diff
@@ -50,8 +51,8 @@ def emojis(message,amount = -1, delete_duplicates = True):
     emoji_table = []
 
     for argument in message.content.split(' '):
-        # TODO: Check if the "word" is a vanilla Discord emoji
-        pass
+        if argument in UNICODE_EMOJI:
+            emoji_table.append(argument)
```
Comment (Bentechy66) [inline interpretation/check.py]: Could @Randium confirm that this is how emoji_table is to be used?

**Comment by Bentechy66** [comment]
Going ahead and merging this, all tests pass, code looks good to me, and I resolved the merge conflicts. Feel free to revert if there are any problems.

**Comment by Randium** [comment]
Hahahah, you merged while I was typing it. Aight, let's delete this branch.

**Pull request merged**